### DISCUSSION TITLE
How do you run the benchmark?

### DISCUSSION THREAD

**Discussion by hsnoil** [leptos-rs/leptos#3430] [Q&A]
I tried running cargo bench but it doesn't seem to run it. No documentation on how to run it either.

Anyone have any idea how to get it to work?

**Comment by gbj** [comment]
Are you talking about the `benchmarks` directory in this repo?

I have not updated it in over a year. It is very out of date and should probably be removed.

**Reply by hsnoil** [reply]
Yes, that one.

If you don't plan to have benchmark comparison, at very least maybe keep a benchmark of leptos itself to keep track of performance improvements/regressions?
